cypress/integration/adminCanCreateTestimonial.feature.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-undef */
+import sizes from '../support/index'
+import TestHelpers from '../support/testhelper'
+
+describe('Admin Can create Testimonials', () => {
+  sizes.forEach((size) => {
+    describe(`and create Testimonials info on ${size}`, () => {
+      beforeEach(() => {
+        cy.intercept('PUT', '**/app_data**', {
+          statusCode: 200,
+          body: {
+            message: 'Testimonial has been created',
+          },
+        })
+        cy.intercept('GET', '**/app_data', {
+          fixture: 'app_data.json',
+        })
+        TestHelpers.sizeParameters(size)
+        cy.visit('/')
+        TestHelpers.authenticate()
+        const selection = 'testimonials'
+        TestHelpers.sizeCase(size, selection)
+      })
+
+      context('successfully', () => {
+        it('is expected to show to show success message on submit', () => {
```

Comment:
change to describe and line 26 'to show to show'

Reply:
fixed